ci-operator/step-registry/cucushift/upgrade/arch-migration/cucushift-upgrade-arch-migration-commands.sh

```diff
@@ -294,7 +294,7 @@ function health_check() {
 
 # Monitor the migration status
 function check_migrate_status() {
-    local wait_migrate="${TIMEOUT}" out avail progress
+    local wait_migrate="${TIMEOUT}" out avail progress arch
     while (( wait_migrate > 0 )); do
```

Comment:
> the [step](https://github.com/openshift/release/blob/master/ci-operator/step-registry/cucushift/upgrade/arch-migration/cucushift-upgrade-arch-migration-commands.sh) might return migrate succeed while migrate is not started.

If I understand the issue here is about "we check that the CV is available and not in progress" when the migration/upgrade is not started yet, right?

Reply:
Right